src/routes/sessions/[id]/+page.svelte

```diff
@@ -221,13 +221,7 @@
 		</div>
 	{/if}
 
-	<PromptEditor
-		bind:model={$session.model}
-		{editor}
-		{handleSubmit}
-		{stopCompletion}
-		{scrollToBottom}
-	/>
+	<PromptEditor bind:session {editor} {handleSubmit} {stopCompletion} {scrollToBottom} />
```

Comment:
At some point we renamed `PromptEditor` to `Prompt` but we didn't update the name.
Can you update the import and the name in on line 224?

```
import PromptEditor from './Prompt.svelte';
```